README.md

```diff
@@ -35,8 +35,12 @@ like to debug Notepad++, but don't have boost.
 ## To build `notepad++.exe`:
 
  1. Open [`PowerEditor\visual.net\notepadPlus.vcxproj`](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/visual.net/notepadPlus.vcxproj)
- 2. Build Notepad++ [like a normal Visual Studio project](https://msdn.microsoft.com/en-us/library/7s88b19e.aspx).
-
+ 2. Copy `SciLexer.dll` from `scintilla\bin\` to the same directory as `notepad++.exe`.
+    - For the `Unicode Release` configuration, the output directory
+      (where `notepad++.exe` is) is `PowerEditor\bin\`.
+    - For the `Unicode Debug` configuration, the output directory
+      (where `notepad++.exe` is) is `PowerEditor\visual.net\Unicode Debug\`.

```

Comment:
Yes, not to build but to execute. It was mentioned under heading "To build SciLexer.dll without boost:"